src/vs/platform/actions/browser/menuItemActionItem.ts

```diff
@@ -142,6 +156,11 @@ export class MenuItemActionItem extends ActionItem {
 		event.preventDefault();
 		event.stopPropagation();
 
+		const altKey = AltKeyEmitter.getInstance(this._contextMenuService);
+		if (altKey.isPressed) {
+			altKey.suppressAltKeyUp();
+		}
```

Comment:
@eamodio Why is this logic only needed in one place and not in the other?

Reply:
@jrieken I'm not sure I'm following &mdash; which other place? Do you mean in `ContextAwareMenuItemActionItem`? If so it might need to be there too, but I can't get any `alt` stuff to work with any context menus &mdash; if you hold `alt` and try to open a context menu it opens and quickly disappears, and if you hit `alt` while a context menu is open it just disappears too.

Reply:
We have two references in the same file... But then I haven't been in this code for long. @isidorn I leave the review to you.